test-validator/src/lib.rs

```diff
@@ -1019,7 +1019,6 @@ impl TestValidator {
                 incremental_snapshot_archives_dir: ledger_path.to_path_buf(),
                 ..SnapshotConfig::default()
             },
-            enforce_ulimit_nofile: false,
```

Comment:
This value is still set to `false` via `ValidatorConfig::default_for_test()` which will then call `BlockstoreOptions::default_for_tests()`